.idea/runConfigurations/Deploy_Robot_and_Remote_Debug.xml

```diff
@@ -4,14 +4,14 @@
     <option name="USE_SOCKET_TRANSPORT" value="true" />
     <option name="SERVER_MODE" value="false" />
     <option name="SHMEM_ADDRESS" />
-    <option name="HOST" value="roborio-1540-frc.local" />
+    <option name="HOST" value="10.15.40.2" />
```

Comment:
This should probably not be hardcoded if possible—the roboRIO is usually, but not always, at that address.